vm/message/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub use signed_message::SignedMessage;
 pub use unsigned_message::UnsignedMessage;
 
 use address::Address;
+
```

Comment:
```suggestion
```

Reply:
I still see the newline in the diff. Did you separate the `use` statements on purpose?

Reply:
No, I didn't